### PR TITLE
Adds Hero Card

### PR DESCRIPTION
**Pull request by bnookala**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -94,11 +96,15 @@ directLine.activity$
               type: "text"
             });
             break;
+          case "application/vnd.microsoft.card.video":
```
Comment (sakkuru) [inline src/index.ts]: "application/vnd.microsoft.card.hero"?

**Review by sakkuru — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 1 above)
Comment (bnookala) [reply src/index.ts]: oops!

Comment (bnookala) [reply src/index.ts]: @sakkuru fixed!!

**Comment by sakkuru** [comment]
LGTM!!

**Review by sakkuru — APPROVED** [review]

**Pull request merged**